sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,35 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server.
+
+  async getGlobalConf(gatewayId, baseURL) {
+
+    let token = this._api._token
+
```

Comment:
```suggestion
const baseUrl = this._stackConfig.gsHost()
if (!baseUrl) {
  throw new Error('Cannot run retrieve "global_conf.json" with disabled component: "gs"')
}
```

Note that `gsHost()` function needs to be added in `sdk/js/util/stack-configuration.js`